support/rag/qa-webservice/private_ai_api.py

```diff
@@ -0,0 +1,109 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+import os
+import nltk
+import psycopg2
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+from sentence_transformers import SentenceTransformer
+import configparser
+from pgvector.psycopg2 import register_vector
+import re
+import requests
+import json
+from openai import OpenAI
+
+# TODO: figure out how to make the parts configurable, i.e. embedding model could be configured here
```

Comment:
It's a chat_api or qa_api? Is that correct ? 

So private_ai_api is not a precise name within the context of the PR.